Source/ThirdParty/libwebrtc/Source/webrtc/api/video_codecs/video_codec.h

```diff
@@ -97,6 +97,21 @@ struct VideoCodecH264 {
   uint8_t numberOfTemporalLayers;
 };
 
+struct VideoCodecH265 {
+  bool operator==(const VideoCodecH265& other) const;
+  bool operator!=(const VideoCodecH265& other) const {
+    return !(*this == other);
+  }
+  bool frameDroppingOn;
+  int keyFrameInterval;
+  const uint8_t* vpsData;
+  size_t vpsLen;
+  const uint8_t* spsData;
+  size_t spsLen;
+  const uint8_t* ppsData;
+  size_t ppsLen;
```

Comment:
is it possible to use a C++ container here instead ?

Reply:
@alfredh Thank you for the review. It is possible to update it but I would like to keep the same coding style as existing codes. Will create separate commit for this code review after this code will be merged.